src/order/directed.lean

```diff
@@ -52,6 +52,8 @@ lemma directed_of_sup [semilattice_sup α] {f : α → β} {r : β → β → Pr
   (H : ∀ ⦃i j⦄, i ≤ j → r (f i) (f j)) : directed r f :=
 λ a b, ⟨a ⊔ b, H le_sup_left, H le_sup_right⟩
 
+alias directed_of_sup ← monotone.directed
```

Comment:
See #7163 - I think this is worth restating to explicitly include "monotone" as one of its arguments

Reply:
You mean that you think I should replace this line with:
```lean
lemma monotone.directed_le [semilattice_sup α] [preorder β] {f : α → β} :
  monotone f → directed (≤) f :=
directed_of_sup
```
pending further revisions in #7163 ?

Reply:
Yes, I think so. I don't feel strongly about `directed` vs `directed_le` as a name.

I think #7163 might be a while - so maybe just add the lemma in that form here, and I'll fix up the conflicts in my PR?

Reply:
Thanks, I'll do that now.